tests/rest/server.go

```diff
@@ -84,12 +98,55 @@ func checkWrongMethodsForEntryPoint() {
 	checkGetEndpointByOtherMethods(apiURL)
 }
 
+// OrganizationsResponse represents response containing list of organizations
+type OrganizationsResponse struct {
+	Organizations []int  `json:"organizations"`
+	Status        string `json:"status"`
+}
+
+func readOrganizationsFromResponse(f *frisby.Frisby) OrganizationsResponse {
+	response := OrganizationsResponse{}
+	text, err := f.Resp.Content()
+	if err != nil {
+		f.AddError(err.Error())
+	} else {
+		json.Unmarshal(text, &response)
+	}
+	return response
+}
+
+// compare two organization lists
+func compareOrglists(a, b []int) bool {
```

Comment:
maybe it's better to use something like `reflect.DeepEqual`?

Reply:
yup, lemme change that